Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -542,32 +555,39 @@ class ArbitraryMipmapDetector
     u32 row_length;
     const u8* buffer;
 
-    PixelRGBAf Sample(u32 x, u32 y) const
+    u32 byte_size() const
```

Comment:
Our function naming conventions are `FunctionName`, the return type can also be `std::size_t`

Reply:
Woops, and I'm used to `pascalCase` for functions anyway. Not sure where that style came from :P

Reply:
Surely you mean `PascalCase`

Reply:
@ZephyrSurfer eeerrrm I meant `camelCase`. :D